Compass/ParseServer.swift

```diff
@@ -25,55 +31,116 @@ class PServer {
        // PFUser.enableAutomaticUser()
         let defaultACL = PFACL()
         defaultACL.hasPublicReadAccess = true
-        
         PFACL.setDefault(defaultACL, withAccessForCurrentUser: true)
-        
     }
     
-    func saveUserLocation(classN: String, uData: UserData) {
+    func logIn(userName: String, pass: String, completition: @escaping (PFUser?, Error?) -> Void) {
+        PFUser.logInWithUsername(inBackground: userName, password: pass) { (user, error) in
+            guard error == nil else {
+                completition (nil, error)
+                return
+            }
+            if user != nil {
+            completition(user, nil)
+            return
+            }
+        }
+    }
+    
+    func SignUp(userName: String, pass: String, completition: @escaping (Bool, Error?) -> Void) {
+        let user = PFUser()
+        user.username = userName
+        user.email = userName
+        user.password = pass
+        user.signUpInBackground { (success, error) in
+            guard error == nil else {
+                completition(false, error)
+                return
+            }
+            if success {
+                completition(true, nil)
+                return
+            }
+            completition(false, MyError.signUpError)
```

Comment:
Oh yeah, you've done it here, great! 👍 
So something similar for login would be good